### ISSUE TITLE
[BUG]: No Batch Selection Available for Product Bundle

### ISSUE THREAD

**Issue by elan-k** [yrestom/POS-Awesome#40]
### Versions

ERPNext: v12.16.2 (version-12)
Frappe Framework: v12.13.0 (version-12)
POS Awesome: v1.0.2 (master)

### Steps to reproduce

1. Make Item with Stock & Batch Enabled (Automatic Batch). Ex: Egg
2. Make Another Item without Stock & Batch / without Inventory. Ex: Egg Pack of 6 Units 
3. Make Product Bundle to link the 1st and 2nd Items. Ex: 1 Egg Pack = 6 Eggs
4. Open POS Awesome and Select 2nd Item (Item Without Inventory). Ex: Select Egg Pack 
5. Try to Save the Invoice by providing Customer & Payment Details.
6. Get Message "Batch number is mandatory for Inventory Item (Egg)" 

### What is Expected?

Option to select the Batch Number for Product Bundle Items as well. 

### What is actually happening?

No option found to mention the batch details and you can't save the invoice without the batch number.

### Screenshots
![No_Batch_for_Product_Bundle_2](https://user-images.githubusercontent.com/78781781/108345283-14d6f700-7204-11eb-8c58-d48481f70278.gif)

**Comment by elan-k** [comment]
When it comes to the Product Bundle, it may hold even multiple items together. So how can we collect the Batch Number for every items used in the Bundle?

At the same time, we can't avoid the Product Bundle option since it is much practical feature nowadays. 

[Auto-Select the Batch on POS](https://github.com/yrestom/POS-Awesome/issues/34) might be the option to resolve this issue in the background irrespective of how many items in the Bundle. 

1. Auto-select if we have a single batch available.
2. Auto-select based on the Expiry Date on the batch if we have multiple batches

In short, we can use the **FIFO** option to resolve this issue.

**Comment by yrestom** [comment]
[feat: auto apply batchs for bundels](https://github.com/yrestom/POS-Awesome/commit/4df99c737e7fab777ede3fa375daa9c4b87e873d)
